services/director-v2/src/simcore_service_director_v2/core/dynamic_services_settings/scheduler.py

```diff
@@ -111,6 +111,11 @@ class DynamicServicesSchedulerSettings(BaseCustomSettings):
         ),
     )
 
+    DYNAMIC_SIDECAR_API_USER_SERVICES_PULLING_TIMEOUT: PositiveFloat = Field(
+        120.0 * _MINUTE,
+        description="before starting the user services pull all the images in parallel",
+    )
```

Comment:
hmm, what is this exactly?
why 120 minutes?
pulling s4l takes at most 5-7 minutes, pulling tip takes at most 15 minutes. why such a high number?

Reply:
I'm unsure on what numbers to use here. I think that docker-cli has no timeout here or a very high one.
We have a 1 hour timeout on states and outputs. I guess we can go for 1 hour here as well.